sway/commands/output/leasing.c

```diff
@@ -0,0 +1,66 @@
+#include "sway/commands.h"
+#include "sway/output.h"
+#include "log.h"
+#include "util.h"
+
+struct cmd_results *output_cmd_leasing(int argc, char **argv) {
+	if (!server.drm_lease_manager) {
+		return cmd_results_new(CMD_FAILURE,
+			"DRM lease manager interface not available");
+	}
+
+	if (!config->handler_context.output_config) {
+		return cmd_results_new(CMD_FAILURE, "Missing output config");
+	}
+	
+	const char *oc_name = config->handler_context.output_config->name;
+	if (strcmp(oc_name, "*") == 0) {
+		return cmd_results_new(CMD_INVALID,
+			"Cannot apply leasing to all outputs");
+	}
+	
+	struct sway_output *sway_output = all_output_by_name_or_id(oc_name);
+	if (!sway_output || !sway_output->wlr_output) {
+		return cmd_results_new(CMD_FAILURE,
+			"Cannot apply leasing to unknown output %s", oc_name);
+	}
+
+	if (argc == 0) {
+		return cmd_results_new(CMD_INVALID, "Missing leasing argument");
+	}
+
+	bool status = true;
+	if (parse_boolean(argv[0], status)) {
```

Comment:
We should be passing `sway_output->leasing` here instead of `status` I think (to make `toggle` work).